src/test/ERC20.t.sol

```diff
@@ -417,7 +431,7 @@ contract ERC20Test is DSTestPlus {
         uint256 amount,
         uint256 deadline
     ) public {
-        deadline = bound(deadline, 0, block.timestamp - 1);
+        hevm.assume(deadline < block.timestamp - 1);
```

Comment:
i still prefer bound to assume, can you revert all the places you used assume instead?